archeryutils/tests/test_rounds.py

```diff
@@ -30,28 +29,22 @@ class TestPass:
     """
 
     def test_init(self) -> None:
-        """
-        Check direct initialisation of a Pass with a target instance.
-        """
+        """Check direct initialisation of a Pass with a target instance."""
         test_pass = Pass(36, _target)
 
         assert test_pass.target == _target
-        assert test_pass.n_arrows == 36
+        assert test_pass.n_arrows == 36  # noqa: PLR2004 Magic value
```

Comment:
Maybe just disable magic value checks at module level, as they are an expected pattern in test code?

Reply:
Done it in the config for all test files given tests will regularly compare to magic numbers.
See https://github.com/jatkinson1000/archeryutils/pull/81/commits/96037e96a4eb396afae3db6588eb196b4d522113.